src/ui/public/notify/Notifier.js

```diff
@@ -93,6 +109,10 @@ define(function (require) {
     return err.stack;
   }
 
+  function infoAsObject(version, buildNum) {
```

Comment:
I was on the fence about it, but I ultimately did it that way given that it was invoked in 3 different places.  I figured this way would mean we're less likely to accidentally introduce inconsistencies in the `info` object setup across the `error`/`warning`/`info` implementations.